Remotion/Data/DomainObjects/Queries/QueryFactory.cs

```diff
@@ -63,10 +66,40 @@ public static class QueryFactory
     public static IQueryable<T> CreateLinqQuery<T> ()
         where T: DomainObject
     {
+      return CreateLinqQuery<T>("<dynamic query>", QueryHelper.EmptyMetadata);
+    }
+
+    /// <summary>
+    /// Creates a <see cref="DomainObjectQueryable{T}"/> used as the entry point to a LINQ query with the default implementation of the SQL 
+    /// generation.
+    /// </summary>
+    /// <typeparam name="T">The <see cref="DomainObject"/> type to be queried.</typeparam>
+    /// <returns>A <see cref="DomainObjectQueryable{T}"/> object as an entry point to a LINQ query.</returns>
+    /// <param name="id">The id that is configured for the create <see cref="IQueryable{T}"/>. The default value is '&lt;dynamic query&gt;'</param>
+    /// <param name="metadata">The metadata that will be configured in the created <see cref="IQueryExecutor"/>. The default value is <see langword="null"/>.</param>
+    /// <example>
+    /// The following example used <see cref="CreateLinqQuery{T}()"/> to retrieve
+    /// an entry point for a query that selects a number of <c>Order</c> objects, filters them by <c>OrderNumber</c>, and orders them by name of
+    /// customer (which includes an implicit join between <c>Order</c> and <c>Customer</c> objects).
+    /// <code>
+    /// var query =
+    ///     from o in QueryFactory.CreateLinqQuery&lt;Order&gt; ()
+    ///     where o.OrderNumber &lt;= 4
+    ///     orderby o.Customer.Name
+    ///     select o;
+    /// var result = query.ToArray();
+    /// </code>
+    /// </example>
```

Comment:
The code example does not fit this override, because it uses the override without parameters.